tests/unit_tests/test_deplete_integrator.py

```diff
@@ -36,15 +37,16 @@ def test_results_save(run_in_tmpdir):
     full_burn_list = []
 
     for i in range(comm.size):
-        vol_dict[str(2*i)] = 1.2
-        vol_dict[str(2*i + 1)] = 1.2
-        full_burn_list.append(str(2*i))
-        full_burn_list.append(str(2*i + 1))
+        vol_dict[str(2 * i)] = 1.2
+        vol_dict[str(2 * i + 1)] = 1.2
+        full_burn_list.append(str(2 * i))
+        full_burn_list.append(str(2 * i + 1))
```

Comment:
PEP8 actually says "[No](https://www.python.org/dev/peps/pep-0008/#other-recommendations)" to this.

Reply:
Reverted these changes